app/decorators/event_decorator.rb

```diff
@@ -0,0 +1,19 @@
+class EventDecorator < Draper::Decorator
+  delegate :picture, :title, :place, :beginning_at, :end_at, :description, :category
```

Comment:
you don't need to delegate methods which you override below like `beginning_at` :)